.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/home/laguan/Proyects/Microverse/JS/TicTacToe-JS/src/js/game.js":"1","/home/laguan/Proyects/Microverse/JS/TicTacToe-JS/src/js/main.js":"2"},{"size":2958,"mtime":1582926546549,"results":"3","hashOfConfig":"4"},{"size":2257,"mtime":1582925122263,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1n263zp",{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/home/laguan/Proyects/Microverse/JS/TicTacToe-JS/src/js/game.js",[],"/home/laguan/Proyects/Microverse/JS/TicTacToe-JS/src/js/main.js",[]]
```

Comment:
You might want to add this file to `.gitignore`.

Reply:
It slipped out of `.gitignore`, sorry!